core/utils.js

```diff
@@ -36,31 +37,21 @@ const style = goog.require('Blockly.utils.style');
 const userAgent = goog.require('Blockly.utils.userAgent');
 
 
-/**
- * Don't do anything for this event, just halt propagation.
- * @param {!Event} e An event.
- */
-const noEvent = function(e) {
-  // This event has been handled.  No need to bubble up to the document.
-  e.preventDefault();
-  e.stopPropagation();
-};
-exports.noEvent = noEvent;
+Object.defineProperty(exports, 'noEvent', {
+  get: function() {
+    deprecation.warn(
```

Comment:
Here and elsewhere in this file, it might be useful to include the 4th arg to warn that specifies what you should use instead.